### ISSUE TITLE
SocketsHttpHandler_HttpClientHandler_Http11_Cancellation_Test.ConnectionFailure_AfterInitialRequestCancelled_SecondRequestSucceedsOnNewConnection fails

### ISSUE THREAD

**Issue by antonfirsov** [dotnet/runtime#93413]
### Error Blob

```json
{
  "ErrorMessage": "ConnectionFailure_AfterInitialRequestCancelled_SecondRequestSucceedsOnNewConnection",
  "BuildRetry": false,
  "ErrorPattern": "",
  "ExcludeConsoleLog": true
}
```

### Reproduction Steps

Started happening on ARM queues on `2023-09-30`.

 ### Known issue validation
**Build: :mag_right:** 
**Result validation: :warning:** Validation could not be done without an Azure DevOps build URL on the issue. Please add it to the "**Build: :mag_right:**" line.
**Validation performed at:** 10/12/2023 7:32:39 PM UTC

#### Summary
|24-Hour Hit Count|7-Day Hit Count|1-Month Count|
|---|---|---|
|0|0|0|

**Comment by liveans** [comment]
@MihaZupan I believe this one is also fixed by #102699. Only failure I've seen from main is 1.5 months ago and it was `TimeoutException`. I'd say we can safely close this.

**Comment by MihaZupan** [comment]
I'm not seeing any more PassedOnRerun entries for this one on non-servicing branches over the last ~month, so you're probably right.
No need to keep tracking it.
